tests/main/nfs-support/task.yaml

```diff
@@ -185,7 +185,7 @@ execute: |
     # - arch: mount.nfs: requested NFS version or transport protocol is not supported
     # - debian-sid: mount.nfs: an incorrect mount option was specified
     # - ubuntu-2*: mount.nfs: an incorrect mount option was specified
-    if not os.query is-arch-linux && not os.query is-debian-sid && [[ "$SPREAD_SYSTEM" != ubuntu-2* ]]; then
+    if not os.query is-arch-linux && not os.query is-debian-sid && not os.query is-debian-11 && [[ "$SPREAD_SYSTEM" != ubuntu-2* ]]; then
```

Comment:
If we end up adding one more exception here, it's probably reasonable to make this a switch statement:
```
case "$SPREAD_SYSTEM" in
    arch-linux-*|debian-sid-*|debian-11-*|ubuntu-2*)
        # nothing
       ;; 
    *)
       ...
       ;;
esac
```